README.md

```diff
@@ -46,10 +46,10 @@ This requires a github token. You can try it out with a github user account
 instead of an organization by replacing the organization argument with
 `--user github_user_name`.
 3. **Register the feedstock with CI services:**
-`conda smithy register-ci --organization conda-forge ./foo-feedstock`.
+`conda smithy register-ci --organization conda-forge --feedstock_directory ./foo-feedstock`.
 This requires tokens for the CI services. You can give the name of a user instead
 of organization with `--user github_user_name`.
-4. Regenerate the feedstock with ``conda smithy regenerate ./foo-feedstock``
+4. Regenerate the feedstock with ``conda smithy rerender --feedstock_directory ./foo-feedstock``
```

Comment:
Might as well change `Regenerate` to `Re-render` if we are changing the CLI call down the line.